tests/integration/docusaurus/deployment_patterns/gcp_deployment_patterns_file_bigquery.py

```diff
@@ -220,98 +228,65 @@
         "Environment Variable GE_TEST_GCP_PROJECT is required to run BigQuery integration tests"
     )
 bigquery_dataset = "demo"
-
 CONNECTION_STRING = f"bigquery://{gcp_project}/{bigquery_dataset}"
 
-# <snippet name="tests/integration/docusaurus/deployment_patterns/gcp_deployment_patterns_file_bigquery_yaml_configs.py datasource_yaml">
-datasource_yaml = rf"""
-name: my_bigquery_datasource
-class_name: Datasource
-execution_engine:
-  class_name: SqlAlchemyExecutionEngine
-  connection_string: bigquery://<GCP_PROJECT_NAME>/<BIGQUERY_DATASET>
-data_connectors:
-   default_runtime_data_connector_name:
-       class_name: RuntimeDataConnector
-       batch_identifiers:
-           - default_identifier_name
-   default_inferred_data_connector_name:
-       class_name: InferredAssetSqlDataConnector
-       name: whole_table
-"""
+# <snippet name="tests/integration/docusaurus/deployment_patterns/gcp_deployment_patterns_file_bigquery.py add_bigquery_datasource">
+datasource = context.sources.add_or_update_sql(
+    name="my_bigquery_datasource",
+    connection_string="bigquery://<GCP_PROJECT_NAME>/<BIGQUERY_DATASET>",
+)
 # </snippet>
 
-# Please note this override is only to provide good UX for docs and tests.
-# In normal usage you'd set your path directly in the yaml above.
-datasource_yaml = datasource_yaml.replace(
-    "bigquery://<GCP_PROJECT_NAME>/<BIGQUERY_DATASET>",
-    CONNECTION_STRING,
+# For tests, we are replacing the `connection_string`
+datasource = context.sources.add_or_update_sql(
+    name="my_bigquery_datasource", connection_string=CONNECTION_STRING
```

Comment:
Can we just use our config substitution for this example?
```
datasource = context.sources.add_or_update_sql(
    name="my_bigquery_datasource",
    connection_string="bigquery://${GCP_PROJECT_NAME}/${BIGQUERY_DATASET}",
)
```

Reply:
HI @Kilo59 I tried this, and was running into trouble with rendering. Will keep the doc as-is. Would that be ok?
![Screen Shot 2023-04-28 at 10 01 41 AM](https://user-images.githubusercontent.com/25670697/235209117-c5b9c151-ef89-4c72-b25a-c59ab7163fa8.png)